src/algebra/geom_sum.lean

```diff
@@ -0,0 +1,132 @@
+/-
+Copyright (c) 2019 Neil Strickland. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Neil Strickland
+
+Sums of finite geometric series
+-/
+
+import algebra.big_operators algebra.commute
+
+universe u
+variable {α : Type u}
+
+open finset
+
+def geom_series [semiring α] (x : α) (n : ℕ) :=
```

Comment:
doc string

Reply:
Oops, missed that one.

Reply:
Remember there's the `#doc_blame` command that will check these things for you.

Reply:
Done.